arm-software/shared/elf2bin/test/test.py

```diff
@@ -0,0 +1,1339 @@
+#!/usr/bin/env python3
```

Comment:
Apologies for being extremely annoying as it would have been a lot of work to put this together.

Is there a reason we couldn't use llvm-lit, yaml2obj and elf2bin and run these like normal lit tests?